addons/mass_mailing_crm/models/mailing_mailing.py

```diff
@@ -36,6 +36,8 @@ def _prepare_statistics_email_values(self):
         values = super(MassMailing, self)._prepare_statistics_email_values()
         if not self.user_id:
             return values
+        if not self.env['crm.lead'].check_access_rights('read', raise_exception=False):
```

Comment:
Actually we are not really leaking information ... Shouldn't we instead compute the field as sudo, as it makes no sense to apply current user ACLs for a statistics email ? We could hide it in view if user does not have the groups (as he cannot do anything with this information, no stat button / no tree view to see), but statistics itself could be computed.

Reply:
Mebbe, out of caution std preferred skipping for now, with the option of modifying it afterwards.